docs/man/git-lfs-config.5.ronn

```diff
@@ -8,6 +8,11 @@ precedence. Most options pertaining to git-lfs are contained in the `[lfs]`
 section, meaning they all named `lfs.foo` or similar, although occasionally an
 lfs option can be scoped inside the configuration for a remote.
 
+These settings are read from a file called `.lfsconfig` found at the root of the
+repository, though legacy support exists to read from `.gitconfig` as well.
+
+The `.lfsconfig` file uses the same format as `.gitconfig`.
```

Comment:
This isn't totally accurate. These settings are read from the normal git config, but can _also_ be read from a `.lfsconfig` file found at the root of the repository. Support for `.gitconfig` at the root of the repository was removed in LFS 2.0.